1/styles/scss/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$mainGray: #7F7F7F;
+$lightGray: #E5E5E5;
+$blue: #22BF95;
```

Comment:
do not attach to color, it can be changed and you will have blue var with red value inside (for example)

Reply:
done